### DISCUSSION TITLE
Add number of 'successful request' to traces list view

### DISCUSSION THREAD

**Discussion by arthurGrigo** [langfuse/langfuse#2030] [Ideas]
### Describe the feature or potential improvement

Would be nice to see how many 'successful request' were made besides 'Usage' and 'Total Cost' for a specific trace.

```python
# ...

with get_openai_callback() as cb:
    completion = chain.invoke(user_input, config= runnable_conf)

print(cb)

#Tokens Used: 512
#	Prompt Tokens: 18
#	Completion Tokens: 494
# Successful Requests: 1    <<<<----
# Total Cost (USD): $0.0010149999999999998

```

**Comment by marcklingen** [comment]
Doesn't this match the number of observations within the trace?

**Reply by arthurGrigo** [reply]
Not if I have a complex langchain.
There could be many intermediate steps between LLM calls which I would see in the observations list of a trace. 

Would be nice to see a single number somewhere which represents how many LLM calls were made.

**Reply by arthurGrigo** [reply]
https://smith.langchain.com/public/a3846fd5-5007-4a50-bbb3-7265325a4034/r


An example for a "complex" prompt chain from LangChains discord.
